net/instaweb/rewriter/cache_extender.cc

```diff
@@ -218,7 +221,8 @@ void CacheExtender::StartElementImpl(HtmlElement* element) {
 
       ResourceSlotPtr slot(driver()->GetSlot(
           input_resource, element, attributes[i].url));
-      Context* context = new Context(this, driver(), NULL /* not nested */);
+      Context* context = new Context(input_role, this, driver(),
+                                     NULL /* not nested */);
```

Comment:
maybe change to nullptr? (there's a couple of candidates below that could also use that)